security/cmd/node_agent_k8s/main.go

```diff
@@ -259,13 +259,33 @@ func setWorkloadCacheTimeParams() {
 		secretRotationJobRunInterval = env
 	}
 
+	// The initial backoff time (in millisec) is a random number between 0 and initBackoff.
+	// Default to 10, a valid range is [10, 120000].
+	var initBackoff int64 = 10
+	env := os.Getenv("INITIAL_BACKOFF")
```

Comment:
Please add _MSEC or something to this env var name

Reply:
Done.